templates/index.html

```diff
@@ -182,7 +182,7 @@ <h1>List-Building Comparison</h1>
 							if (data[idx]['title'] == '-') {
 								var wpurl = 'Article missing';
 							} else {
-								var wpurl = '<a href="https://' + document.getElementById('lang').value + '.wikipiedia.org/wiki/' + data[idx]['title'] + '">' + data[idx]['title'].replaceAll('_', ' ') + '</a>';
+								var wpurl = '<a href="https://' + document.getElementById('lang').value + '.wikipiedia.org/wiki/' + data[idx]['title'].replaceAll(' ', '_') + '">' + data[idx]['title'].replaceAll('_', ' ') + '</a>';
```

Comment:
Can you fix the fact that i also spelled Wikipedia wrong in the URL? Not related to anything you did but seems convenient to include it in this patch. Thanks!

Reply:
how did I miss that.
fixed now.